Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Builds basic gips docker image; used as a foundation or basis.
+
+FROM geographica/gdal2:latest
+
+COPY . /gips
+WORKDIR /gips
+
+# TODO not clear if gfortran is needed when we're not compiling sixs after all
+RUN apt-get -y update \
+    && apt-get install -y \
+       python3-pip \
+       curl \
+       wget \
+       gfortran \
+       libgnutls28-dev
+
+# we don't have to do this; can copy a binary into place instead
```

Comment:
Here's my dev build & dev loop on this python 3 effort in case it'd be helpful:

Presently I use `docker/gips-test-full.docker` for dev purposes, which contains sixs and can pass the tests that require it.  This is how I do a full docker build:

```
docker build --no-cache -t gips-base -f Dockerfile .
docker build --no-cache -t gips-ci -f docker/gips-ci.docker .
docker build --no-cache -t gips-test-full -f docker/gips-test-full.docker .
```

Then most of my time in the dev loop is spent running pytest eg:

```
alias r='docker run --rm -i -v /home/tolson/src/gips/:/gips -v /home/tolson/src/gips-sys-test-assets:/artifact-store'
# unit & integration tests:
time r gips-test-full pytest -r es -s -vv --setup-repo -k 'unit or int'
# system tests for landsat, say:
time r gips-test-full pytest -r es --slow --sys -s -vv --setup-repo --stepwise -k 'sys and landsat'
```

Reply:
PS I was feeling documenty so I added more expansive instructions to `docker/README.md` which I'll push today.

Reply:
> Here's my dev build & dev loop
Cool, thanks, that confirms that what I'm doing is not different from you, which is totally great. Just for symmetry should the gips/Dockerfile I made be moved into docker/ directory at some point?

Reply:
Any time.  I suppose it comes down to whether it's better to follow docker convention and put a `Dockerfile` in a project's root directory vs. keep everything in the same place for consistency.  I can't see a clear advantage either way.